domain/entities/transfers/transfers.go

```diff
@@ -0,0 +1,36 @@
+package transfers
+
+import (
+	"errors"
+	"time"
+)
+
+var (
+	errInvalidLengthID = errors.New("Empty ID not allowed")
+	errInvalidAmount   = errors.New("Invalid amount, be must > 0")
+	errEqualAccounts   = errors.New("Equal source and destination accounts are not allowed.")
+)
+
+type Transfer struct {
+	ID                     string    `json:"id"`
+	Account_origin_ID      string    `json:"account_origin_id"`
```

Comment:
```suggestion
	AccountOriginID      string    `json:"account_origin_id"`
```
Geralmente usamos camel case pra fields de struct (ou variáveis em geral, na verdade). A JSON  tag não tem problema.

Reply:
correção realizada.

Reply:
Acho que faltou o `Created_At`